### DISCUSSION TITLE
请求全面收录 fcitx5 等上游设施

### DISCUSSION THREAD

**Discussion by yanhuacuo** [linuxdeepin/developer-center#4668] [General | 常规讨论]
今日，又于论坛见 [反馈98五笔码表未被收录](https://bbs.deepin.org/zh/post/257924?offset=0&postId=1476088) ，其实这并不准确。

如所见，两年前 [fcitx5-table-extra](
https://github.com/fcitx/fcitx5-table-extra) 这个项目就已收录了98五笔码表，以及仓颉、郑码等。

https://github.com/fcitx/fcitx5-table-extra/tree/master/tables

Manjaro 已收录：

https://software.manjaro.org/package/fcitx5-table-extra

Fedora 已收录：

https://packages.fedoraproject.org/pkgs/fcitx5-table-extra/

中文发行版对输入法的依赖倾向更大，DeepinV23 默认 Fcitx5 也是长足进步，希望早日收录这个软件包，对国内形码用户意义重大。

因为……太多人并不了解上游资源的建设情况。

**Comment by BLumia** [comment]
cc @zsien 有计划打包吗？

不过和之前的讨论一样，如果你有兴趣与意愿，强烈建立直接参与到软件包的维护中来 :)

**Reply by zsien** [reply]
等我下周打一个

**Reply by yanhuacuo** [reply]
> 等我下周打一个

蹭个车 ： https://github.com/yanhuacuo/rime98wb

一个基于中州韵、开源的「98五笔中文输入法套件」，并自带百万级词库处理性能的全盘可视化管理工具。

包名：rime98wb

**Reply by shenmo7192** [reply]
better deepin23已收录

**Comment by shenmo7192** [comment]
已收录
https://bbs.deepin.org/post/257738

**Comment by xzl01** [comment]
> 等我下周打一个
是否打了，不然我来抢活了
